gtest_parallel_tests.py

```diff
@@ -143,6 +159,35 @@ def test_run_task_basic(self):
 
     self.assertEqual(task_manager.global_exit_code, 1)
 
+  @unittest.skipIf(sys.platform == 'win32', 'non-Windows specific')
```

Comment:
I would expect you to mock out sys.platform (similar to os.environ above), so that we can run windows tests even when not on a Windows platform. I don't expect anyone to run these tests on Windows, so it's good to still have the same test coverage. Does that make sense?